dev/com.ibm.ws.transaction/resources/com/ibm/ws/transaction/services/TransactionMessages.nlsprops

```diff
@@ -63,4 +62,9 @@ UNEXPECTED_TRAN_ERROR.useraction=Check the logs for previous errors or FFDCs.
 # {0} is the TransactionRolledbackException thrown
 GLOBAL_TRAN_ROLLBACK=CWLIB0003E: Global transaction rolled-back due to timeout: {0}
 GLOBAL_TRAN_ROLLBACK.explanation=The current global transaction has timed out and therefore has been rolled-back.
-GLOBAL_TRAN_ROLLBACK.useraction=Ensure the global transaction finishes within the allowed time frame or increase the timeout.
\ No newline at end of file
+GLOBAL_TRAN_ROLLBACK.useraction=Ensure the global transaction finishes within the allowed time frame or increase the timeout.
```

Comment:
ID edit:
GLOBAL_TRAN_ROLLBACK.useraction=Ensure that the global transaction finishes within the allowed time frame, or increase the timeout.